pkgs/development/python-modules/fuzzywuzzy/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, python, fetchFromGitHub, python-Levenshtein, pycodestyle, hypothesis, pytest }:
+
+buildPythonPackage rec {
+  pname = "fuzzywuzzy";
+  version = "0.16.0";
```

Comment:
I'm seeing https://github.com/seatgeek/fuzzywuzzy/releases/tag/0.17.0 on their github. 
I think we should use that version.

Reply:
Done